tar.go

```diff
@@ -44,20 +47,31 @@ type tarStream struct {
 }
 
 func (ts *tarStream) readHeaders() {
+	// remove "time" keyword
+	notimekws := []string{}
+	for _, kw := range ts.keywords {
+		if !inSlice(kw, notimekws) {
+			if kw != "time" {
+				notimekws = append(notimekws, kw)
+			} else {
+				if !inSlice("tar_time", ts.keywords) {
```

Comment:
this is not different. Are you meaing `} else if kw == "time" && !inSlice("tar_time", ts.keywords) {`?